runhouse/servers/cluster_servlet.py

```diff
@@ -232,7 +243,9 @@ async def asave_status_metrics_to_den(self, status: dict):
 
         status_data = {
             "status": ResourceServerStatus.running,
-            "resource_type": status_copy.get("cluster_config").pop("resource_type"),
+            "resource_type": status_copy.get("cluster_config").pop(
```

Comment:
why wouldn't resource type be included?

Reply:
because according to the status schema, it is saved outside of the cluster config, so it is redundant to save it twice.